modules/3d-tiles/src/parsers/parse-3d-tile-point-cloud.js

```diff
@@ -36,8 +36,6 @@ function extractPointCloud(tile) {
     throw new Error('POINTS_LENGTH must be defined');
   }
 
-  // const batchTable = new Tile3DBatchTable(tile);
```

Comment:
Point clouds do have batch tables: https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/specification/TileFormats/PointCloud#layout

We'll need to add this back soon.